python/cugraph/cugraph/datasets/dataset.py

```diff
@@ -183,7 +186,15 @@ def get_edgelist(self, download=False):
             header = None
             if isinstance(self.metadata["header"], int):
                 header = self.metadata["header"]
-            self._edgelist = cudf.read_csv(
+            if create_using is None:
+                reader = cudf
```

Comment:
If applying the above suggestion:
```suggestion
                reader = cudf.read_csv
```